### ISSUE TITLE
[Loki] Error: create: failed to create: Request entity too large: limit is 3145728

### ISSUE THREAD

**Issue by nick-oconnor** [grafana/helm-charts#3238]
On upgrading my standalone Loki chart from 6.7.1 to 6.7.2, helm fails with:
```
Error: create: failed to create: Request entity too large: limit is 3145728
```
I'm running helm via terraform plugin. The error is [usually thrown](https://github.com/helm/helm/issues/7264#issuecomment-567373763) when a binary file is accidentally included in the chart.

**Comment by Cardes** [comment]
I think the file in question is this ~13Mi sized one:

`loki\src\helm-test\helm-test`

**Comment by z0rc** [comment]
Upstream reported at https://github.com/grafana/loki/issues/13633.

**Comment by crazyelectron-io** [comment]
Stumbled upon the same error when trying to install Loki with
```shell
helm upgrade --install --values loki-values.yaml loki --namespace=logging grafana/loki
```
Adding `--version 6.7.3` solved it for me.

**Comment by nick-oconnor** [comment]
Closing as this issue has been resolved in Loki v6.7.3. Thanks!
